docs/reference/filebeat/filebeat-input-filestream.md

```diff
@@ -325,53 +325,88 @@ The setting relies on the modification time of the file to determine if a file i
 
 To remove the state of previously harvested files from the registry file, use the `clean_inactive` configuration option.
 
-## Take over [filebeat-input-filestream-take-over]
+#### `take_over` [filebeat-input-filestream-take-over]
+
+```{applies_to}
+stack: beta
+```
+
+::::{warning}
+The take over mode is still in beta, however, it should be generally safe to use.
+::::
```

Comment:
I _think_ this is still in beta. Can you confirm?

Reply:
Yes, it is still in beta.